addons/mail/static/src/js/tours/mail.js

```diff
@@ -22,9 +22,12 @@ tour.register('mail_tour', {
         actions.text("SomeChannel_" + t, this.$anchor.find("input"));
     },
 }, {
-    trigger: ".ui-autocomplete .ui-menu-item > a:contains(Private)",
-    content: _t("<p> Create a private channel.</p>"),
-    position: 'bottom',
+    trigger: ".ui-autocomplete:has(a:contains(Create))",
```

Comment:
I think this selector is too weak, "Create" string can be translated it other languages, selector must not rely on translatable text.